frontend/src/pages/chat/components/chat-footer/chat-footer.tsx

```diff
@@ -0,0 +1,56 @@
+import { Button, Input } from '#libs/components/components.js';
+import { Icon } from '#libs/components/icon/icon.js';
+import { DEFAULT_INPUT } from '#libs/constants/constants.js';
+import { typer } from '#libs/helpers/helpers.js';
+import { useAppForm, useCallback } from '#libs/hooks/hooks.js';
+import { type FormControl, type FormFieldValues } from '#libs/types/types.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  onSend: ({ text }: { text: string }) => void;
+};
+
+type HandleForm = () => void;
+
+type FormValue = {
+  text: string;
+} & FormFieldValues;
+
+const ChatFooter: React.FC<Properties> = ({ onSend }) => {
+  const { control, handleSubmit, reset } = useAppForm<FormValue>(DEFAULT_INPUT);
+
+  const onSubmit = useCallback(
+    ({ text }: FormValue): void => {
+      onSend({ text });
+      reset();
+    },
+    [onSend, reset],
+  );
+
+  return (
+    <footer className={styles['chat-footer']}>
+      <form
+        className={styles['form']}
+        onSubmit={handleSubmit(onSubmit) as HandleForm}
+      >
+        <Input
+          placeholder="Type a message"
+          style="chat-input"
+          autoComplete="off"
+          required
+          name="text"
+          control={typer<
+            FormControl<FormValue, null>,
+            FormControl<FormFieldValues, null>
+          >(control)}
```

Comment:
```suggestion
          control={control}
```
Remove `typer` helper